### PR TITLE
Feature/sa 13 add lambdas

### PR DESCRIPTION
**Pull request by blackmamo**

### CODE REVIEW

File: infrastructure/infrastructure.tf [hunk 1]
```diff
@@ -40,26 +49,66 @@ provider "aws" {
...
+  results_bucket_arn = "${module.nmap_task.results_bucket_arn}"
+  results_bucket_id  = "${module.nmap_task.results_bucket_id}"
+  ssm_source_stage   = "${local.ssm_source_stage}"
+}
+
+data "aws_ssm_parameter" "vpc_id" {
+  name = "/${var.app_name}/${local.ssm_source_stage}/vpc/id"
+}
+
+resource "aws_security_group" "outbound_only" {
```
Comment (progers-tech) [inline infrastructure/infrastructure.tf]: use the security group in task execution

File: serverless.yml [hunk 2]
```diff
@@ -36,14 +39,24 @@ custom:
     # Dont use zip, it makes things bigger because the no deploy filtering seems to be ignored
     zip: false
     noDeploy:
+      # provided by our layer
       - utils
-      - sec_an_shared*
+      # the rest by lambda
```
Comment (progers-tech) [inline serverless.yml:44]: check if these are added automatically?

**Review by progers-tech — DISMISSED** [review]

File: serverless.yml (hunk 2 above)
Comment (blackmamo) [reply serverless.yml:44]: They are not :(

File: infrastructure/infrastructure.tf (hunk 1 above)
Comment (blackmamo) [reply infrastructure/infrastructure.tf]: done

**Pull request merged**